.bingo/variables.env

```diff
@@ -1,7 +1,7 @@
 # Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.2.3. DO NOT EDIT.
 # All tools are designed to be build inside $GOBIN.
 # Those variables will work only until 'bingo get' was invoked, or if tools were installed via Makefile's Variables.mk.
-gobin=$(go env GOBIN)
+local gobin=$(go env GOBIN)
```

Comment:
Same here, `local` should be omitted, because it works only in bash/shell functions. The Fix for this is https://github.com/bwplotka/bingo/pull/47